website_rma/views/res_config_settings_views.xml

```diff
@@ -16,7 +16,6 @@
                         title="Set the default RMA Team and RMA responsible of the RMAs created from the Request RMA page."
                     >
                         <div class="o_setting_right_pane">
-                            <label for="id" string="Request RMA Form" />
```

Comment:
Why removing this?

Reply:
I got such error with it:
Name or id 'id' used in 'label for' must be present in view but is missing.

Reply:
Surely it was a trick for avoiding the no `for` warnings in the `label` tag. I tried this and makes the same effect.
```suggestion
                            <label for="rma_default_team_id" string="Request RMA Form" />
```